### ISSUE TITLE
Not a Class?

### ISSUE THREAD

**Issue by jgphilpott** [maximeq/three-js-capsule-geometry#5]
I like this but should it not be a class rather than a function ... like all other geometries in three.js?

**Comment by maximeq** [comment]
You're right, we migrated to three 122 but did not moved to class syntax and are still using the old way of defining classes.

It shiould not impact usage though, you can still do:
`new CapsuleBufferGeometry(...);`

Feel free to PR :)

**Comment by jgphilpott** [comment]
Okay, I don't know if I will put in a PR now cuz im a little busy but maybe sometime in the future. Your capsule is even better than the one in the main three.js library cuz it supports more parameters.

Thanks!
